Gemfile.lock

```diff
@@ -266,4 +266,4 @@ DEPENDENCIES
   xcpretty-travis-formatter!
 
 BUNDLED WITH
-   1.17.3
+   2.0.2
```

Comment:
Should this change be made separately or is there a reason for this to happen now?

Reply:
`gem install bundler` will by default install Bundler 2 if we update ruby, which I think we have to do because of the CircleCI image. I figure given that fact, it makes sense to do this now, since we're in the same building with folks for a couple of days to help with fixes if needed? 😅

Reply:
I guess, and we definitely want to upgrade but this seems like the kind of change which could cause issues 🤔

Anyway, if you’re comfortable with it, let’s go ahead.